llvm/docs/Contributing.rst

```diff
@@ -122,28 +121,9 @@ For more information on LLVM's code-review process, please see :doc:`CodeReview`
 For developers to commit changes from Git
 -----------------------------------------
 
-Once a patch is reviewed, you should rebase it, re-test locally, and commit the
-changes to LLVM's main branch. This is done using `git push` if you have the
-required access rights. See `committing a change
-<Phabricator.html#committing-a-change>`_ for Phabricator based commits or
-`obtaining commit access <DeveloperPolicy.html#obtaining-commit-access>`_
-for commit access.
-
-Here is an example workflow using git. This workflow assumes you have an
-accepted commit on the branch named `branch-with-change`.
-
-.. code-block:: console
-
-  # Pull changes from the upstream main branch.
-  % git checkout main && git pull
-  # Rebase your change onto main.
-  % git rebase --onto main --root branch-with-change
-  # Rerun the appropriate tests if needed.
-  % ninja check-$whatever
-  # Check that the list of commits about to be pushed is correct.
-  % git log origin/main...HEAD --oneline
-  # Push to Github.
-  % git push origin HEAD:main
```

Comment:
L165 below talks about a pre-push hook; but the diff removed above removes the recommended workflow about `git push` and the new Github doc recommends using `gh pr merge`.

Two thoughts:
1. Does the pre-push hook still run if using the `gh` CLI?  If not, we might need to clean that up.
2. Can we link to the Github page's section on merging "Landing your change" from this page's "For developers to commit changes from Git" section?